sdk/monitor/monitor-opentelemetry-exporter/samples-dev/utils/helloworld_grpc_pb.js

```diff
@@ -0,0 +1,64 @@
+// GENERATED CODE -- DO NOT EDIT!
```

Comment:
We might need to do some special handling for these generated JS files. I'll clone this PR and look into it. At the very least we'll need to decide either to strongly-type these models or use them as `any` in the consuming samples.